src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -78,7 +78,7 @@ public CaseDetails createCase(String authorization, String caseType, String even
         var ccdCase = ccdApiClient.startForCaseworker(
             authorization,
             s2sToken,
-            userDetails.getId(),
+            "123456",
```

Comment:
Can you change this to userDetails.getId() ?

Reply:
Fixed